src/Microsoft.AspNetCore.Sockets.Http/Internal/Transports/WebSocketsTransport.cs

```diff
@@ -143,7 +144,7 @@ public WebSocketsTransport(WebSocketOptions options, IDuplexPipe application, De
 #if NETCOREAPP2_1
                     var receiveResult = await socket.ReceiveAsync(memory, CancellationToken.None);
 #else
-                    var isArray = memory.TryGetArray(out var arraySegment);
+                    var isArray = MemoryMarshal.TryGetArray<byte>(memory, out var arraySegment);
```

Comment:
Is the `<byte>` not inferrable?

Reply:
The first parameter type is `ReadOnlyMemory`, it's either inference or implicit conversion, unfortunately.